<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>YlzSqlViewerClass</class>
 <widget class="QMainWindow" name="YlzSqlViewerClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>594</width>
    <height>384</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>YlzSqlViewer</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="QMdiArea" name="mdiArea"/>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>594</width>
     <height>18</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionMonitor"/>
   <addaction name="actionDatabase"/>
   <addaction name="actionDDL"/>
   <addaction name="separator"/>
   <addaction name="actionSettings"/>
   <addaction name="separator"/>
   <addaction name="actionLogDownload"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionStart">
   <property name="icon">
    <iconset resource="YlzSqlViewer.qrc">
     <normaloff>:/YlzSqlViewer/Resources/start.ico</normaloff>
     <normalon>:/YlzSqlViewer/start.png</normalon>:/YlzSqlViewer/Resources/start.ico</iconset>
   </property>
   <property name="text">
    <string>Start</string>
   </property>
   <property name="toolTip">
    <string>Start</string>
   </property>
  </action>
  <action name="actionStop">
   <property name="icon">
    <iconset resource="YlzSqlViewer.qrc">
     <normaloff>:/YlzSqlViewer/Resources/stop.ico</normaloff>:/YlzSqlViewer/Resources/stop.ico</iconset>
   </property>
   <property name="text">
    <string>Stop</string>
   </property>
   <property name="toolTip">
    <string>Stop</string>
   </property>
  </action>
  <action name="actionClear">
   <property name="icon">
    <iconset resource="YlzSqlViewer.qrc">
     <normaloff>:/YlzSqlViewer/Resources/clear.ico</normaloff>:/YlzSqlViewer/Resources/clear.ico</iconset>
   </property>
   <property name="text">
    <string>Clear</string>
   </property>
   <property name="toolTip">
    <string>Clear</string>
   </property>
  </action>
  <action name="actionSettings">
   <property name="icon">
    <iconset resource="YlzSqlViewer.qrc">
     <normaloff>:/YlzSqlViewer/Resources/settings.ico</normaloff>:/YlzSqlViewer/Resources/settings.ico</iconset>
   </property>
   <property name="text">
    <string>Settings</string>
   </property>
   <property name="toolTip">
    <string>Settings</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="YlzSqlViewer.qrc">
     <normaloff>:/YlzSqlViewer/Resources/save.ico</normaloff>:/YlzSqlViewer/Resources/save.ico</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
   <property name="toolTip">
    <string>Save</string>
   </property>
  </action>
  <action name="actionLogDownload">
   <property name="icon">
    <iconset resource="YlzSqlViewer.qrc">
     <normaloff>:/YlzSqlViewer/Resources/log_download.ico</normaloff>:/YlzSqlViewer/Resources/log_download.ico</iconset>
   </property>
   <property name="text">
    <string>LogDownload</string>
   </property>
   <property name="toolTip">
    <string>LogDownload</string>
   </property>
  </action>
  <action name="actionDatabase">
   <property name="icon">
    <iconset resource="YlzSqlViewer.qrc">
     <normaloff>:/YlzSqlViewer/Resources/database.ico</normaloff>:/YlzSqlViewer/Resources/database.ico</iconset>
   </property>
   <property name="text">
    <string>Database</string>
   </property>
   <property name="toolTip">
    <string>Database</string>
   </property>
  </action>
  <action name="actionMonitor">
   <property name="icon">
    <iconset resource="YlzSqlViewer.qrc">
     <normaloff>:/YlzSqlViewer/Resources/monitor.ico</normaloff>:/YlzSqlViewer/Resources/monitor.ico</iconset>
   </property>
   <property name="text">
    <string>Monitor</string>
   </property>
   <property name="toolTip">
    <string>Monitor</string>
   </property>
  </action>
  <action name="actionDDL">
   <property name="icon">
    <iconset resource="YlzSqlViewer.qrc">
     <normaloff>:/YlzSqlViewer/Resources/ddl.ico</normaloff>:/YlzSqlViewer/Resources/ddl.ico</iconset>
   </property>
   <property name="text">
    <string>DDL</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="YlzSqlViewer.qrc"/>
 </resources>
 <connections/>
</ui>
